src/GlassCat/utilities.jl

```diff
@@ -318,3 +318,28 @@ function plot_indices(glass::AbstractGlass; polyfit::Bool = false, fiterror::Boo
 
     gui(p)
 end
+
+
+""" Draw a scatter plot of index vs dispersion (the derivative of index with respect to wavelength). Both index and dispersion are computed at wavelength λ. If showprefixglasses is true then glasses with names like F_BAK7 will be displayed. Otherwise glasses that have a leading letter prefix followed by an underscore, such as F_, will not be displayed."""
+function drawglassmap(glasscatalog::Module; λ = 550nm, glassfontsize = 3, showprefixglasses = false)
+    wavelength = Float64(ustrip(uconvert(u"μm", λ)))
+    indices = Vector{Float64}(undef,0)
+    dispersions = Vector{Float64}(undef,0)
+    glassnames = Vector{String}(undef,0)
+
+    for name in names(glasscatalog)
+        glass = eval(:($glasscatalog.$name))
+        glassstring = String(name)
+        hasprefix = occursin("_",glassstring)
+        
+        if typeof(glass) !== Module && index(glass,wavelength) > 0 && index(glass,wavelength) < 3 && (showprefixglasses ? true : !hasprefix)
```

Comment:
Can this be simplified or explained by assigning it to an intermediate variable? I think it's quite hard to understand at first.